analyzer/codechecker_analyzer/cmd/check.py

```diff
@@ -318,19 +318,21 @@ def add_arguments_to_parser(parser):
     analyzer_opts.add_argument('--config',
                                dest='config_file',
                                required=False,
-                               help="Allow the configuration from an explicit "
-                                    "JSON based configuration file. The "
-                                    "value of the 'analyzer' key in the "
+                               help="R|Allow the configuration from an "
+                                    "explicit JSON based configuration file. "
+                                    "The value of the 'analyzer' key in the "
                                     "config file will be emplaced as command "
                                     "line arguments. The format of "
-                                    "configuration file is: "
-                                    "{"
-                                    "  \"analyzer\": ["
-                                    "    \"--enable=core.DivideZero\","
-                                    "    \"--enable=core.CallAndMessage\","
-                                    "    \"--clean\""
-                                    "  ]"
-                                    "}.")
+                                    "configuration file is:\n"
```

Comment:
These new line characters have no effect for me.

Reply:
Are you sure that you have rebuilt the CodeChecker package? For me it looks like this:
![image](https://user-images.githubusercontent.com/6695818/85985795-83ecaa80-b9eb-11ea-9ac2-1c5db0fc4a02.png)
@gyorb Could you please try this out too?

Reply:
It looks like this for me in zsh:
```
  --config CONFIG_FILE  Allow the configuration from an explicit JSON based                    
                        configuration file. The value of the 'analyzer' key in 
                        the config file will be emplaced as command line                       
                        arguments. The format of configuration file is:         
                        {                                                                      
                          "analyzer": [                                                        
                            "--enable=core.DivideZero",                         
                            "--enable=core.CallAndMessage",                                    
                            "--report-hash=context-free-v2",                
                            "--verbose=debug",                                                 
                            "--clean"                                                          
                          ]                                                                    
                        } (default: None)  
```